index.html

```diff
@@ -754,6 +755,11 @@
   <ol class='algorithm'>
       <li id="step-create-object">Create a new <a>PerformanceResourceTiming</a> object and
       set <a href="#entryType-attribute">entryType</a> to the DOMString <code>resource</code>.
+        <ul>
+            <li>
+                If the fetch is aborted for any reason, immediately go to step 18 (queue the record).

```

Comment:
~"For the remaining steps, if the fetch is...." - I think we need to clarify that this applies for all following steps.

... aborted due to a _network error_ (same ref as above)